src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -1827,6 +1827,13 @@ public void regenerateCertificatesOf(Consumer consumer, boolean lazy) {
         this.ecGenerator.regenerateCertificatesOf(consumer, lazy);
     }
 
+    @Override
+    public void regenerateCertificatesOf(Consumer consumer, List<String> updatedEnvironments,
+        List<String> environmentChangeList, boolean lazy) {
+        this.ecGenerator.regenerateCertificatesOf(consumer, updatedEnvironments,
+            environmentChangeList, lazy);
+    }
```

Comment:
follow up from my other comment: if we break out the entitlement filtering functionality to a separate class, this method is also not needed